cumulusci/robotframework/tests/salesforce/pageobjects/ObjectManagerPageObject.py

```diff
@@ -0,0 +1,144 @@
+		self.salesforce.populate_field('Field Label', field_name)
+		self.selenium.wait_until_page_contains_element(checkbox_option,60)
+		self.selenium.click_element(checkbox_option)
+		self.selenium.click_element(next_button)
+		self.selenium.wait_until_page_contains_element(formula_txtarea,60)
+		self.selenium.get_webelement(formula_txtarea).send_keys(formula)
+		self.selenium.click_element(check_syntax)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(save_button)
+		self.selenium.wait_until_location_contains("FieldsAndRelationships/view", timeout=90,
+												   message="Detail page did not load in 1 min")
+	
+	def create_lookup_field(self,field_name, related):
+		"""Creates a Lookpup field by taking in the inputs field_name and related field"""
+		lookup_locator = object_manager['input'].format("dtypeY")
+		next_button = object_manager['button'].format("Next")
+		save_button = object_manager['button'].format("Save")
+		option = object_manager['select_related_option'].format(related)
+		field_label = object_manager['input'].format("MasterLabel")
+		related = object_manager['select_related'].format("DomainEnumOrId")
+		self.selenium.wait_until_page_contains_element(lookup_locator,60)
+		self.selenium.click_element(lookup_locator)
+		time.sleep(1)
+		self.selenium.click_element(next_button)
+		self.selenium.wait_until_page_contains_element(related,60)
+		self.selenium.scroll_element_into_view(related)
+		self.selenium.get_webelement(related).click()
+		self.selenium.click_element(option)
+		time.sleep(2)
+		self.selenium.click_element(next_button)
+		self.salesforce.populate_field('Field Label', field_name)
+		self.salesforce.populate_field('Description', "This is a custom field generated during automation")
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(next_button)
+		self.selenium.click_element(save_button)
+		self.selenium.wait_until_location_contains("FieldsAndRelationships/view", timeout=90,
+												   message="Detail page did not load in 1 min")
+
+	@capture_screenshot_on_error
+	def create_custom_field(self, **kwargs):
+		"""Ensure that the custom field does not exist prior and Creates a custom field based on type paramenter and the field_name
+		   IF the custom field exists it will not create the custom field and exits out of object manager
+		 """
+		search_button = object_manager['input'].format("globalQuickfind")
+		self.selenium.wait_until_page_contains_element(search_button,60)
+		self.selenium.get_webelement(search_button).send_keys(kwargs['Field_Name'])
+		self.selenium.get_webelement(search_button).send_keys(Keys.ENTER)
+		time.sleep(1)
+		self.salesforce.wait_until_loading_is_complete()
+		search_results = object_manager['search_result'].format(kwargs['Field_Name'])
+		count = len(self.selenium.get_webelements(search_results))
+		if not count == 1:
```

Comment:
I think this would be slightly easier to read as `if count != 1`

Reply:
Done